tests/docs/running-perf-tests.md

```diff
@@ -255,4 +255,8 @@ The Prometheus installation above comes with a pushgateway.
 
 * [Grafana Dashboard for Perf Test](../config/grafana-perf-test-dashboard.json)
   
-  On running the perf-tests now, the metrics are collected from pushgateway by Prometheus and is made available for visualization as a dashboard by importing the above template in Grafana. 
+  On running the perf-tests now, the metrics are collected from pushgateway by Prometheus and is made available for visualization as a dashboard by importing the above template in Grafana.
+
+### Dashboard for service_invocation_http
+<img width="1267" alt="grafana_dashboard" src="https://github.com/SushmitaSeth/dapr/assets/106324338/fefbf1af-a129-49e7-a6c2-c74536514ed6">
```

Comment:
This image should not come from your repo. Please move it to https://github.com/dapr/dapr/tree/master/tests/docs/img/perf_dashboard.png

Reply:
Changed